tests/main/snap-run-gdbserver/task.yaml

```diff
@@ -11,12 +11,12 @@ restore: |
     apt autoremove -y gdbserver gdb
 
 execute: |
-    echo "Test snap run --experimental-gdbserver works"
+    echo "Test snap run --gdbserver works"
 
     # run the gdbserver command as a user
     # XXX: use "systemd-run --user -p StandardOutput=file:$(pwd)/stdout"
     # shellcheck disable=SC2016
-    su -c 'snap run --experimental-gdbserver test-snapd-sh.sh -c "echo hello-hello-hello uid:$UID"' test >stdout &
+    su -c 'snap run --gdbserver test-snapd-sh.sh -c "echo hello-hello-hello uid:$UID"' test >stdout &
```

Comment:
We should probably do this basic test with --experimental-gdbserver and --gdbserver (maybe via environment flag) for as long as we'll have the flag in `snap run`.